eth/backend.go

```diff
@@ -42,19 +43,21 @@ type Backend interface {
 type backend struct {
 	*ethclient.Client
 	methods      map[string]string
```

Comment:
Do we still need this map if we're storing a map from method IDs -> ABIs? If we have access to the ABI then we can fetch the method name via `abi.MethodById(methodID).Name`.

Reply:
fixed in 8a3dda8